build_debian.sh

```diff
@@ -260,6 +260,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT umount /proc
 ## Prepare empty directory to trigger mount move in initramfs-tools/mount_loop_root, implemented by patching
 sudo mkdir $FILESYSTEM_ROOT/host
 
+## Create loop device for /var/log to limit its size to 1000000*512=500MB
+sudo mkdir -p $FILESYSTEM_ROOT/var/disk-img
+sudo LANG=C chroot $FILESYSTEM_ROOT dd if=/dev/zero of=/var/disk-img/var-log.ext4 count=1000000
```

Comment:
for a platform that has only 2G disk. after installation we use about 800M. If we do upgrade, we need to download another image which is about 400MB. We may be some space for further dockers. Perhaps 256M logs for those platforms. For a device with 16G or more disk, we would like allocate more disk for the logs. Perhaps 4G should be good.  Maybe default is 4G and we need to specify values for some other platform.